src/components/header_menu.styl

```diff
@@ -1,8 +1,11 @@
 .header-menu
     align-items center
+    box-shadow 0px 0px 0px 0.5px rgba(29, 33, 42, 0.12), 0px 2px 4px rgba(29, 33, 42, 0.0793047), 0px 4px 16px rgba(29, 33, 42, 0.06)
     display flex
     justify-content space-between
     padding 0.5rem 1rem
+    position relative
+    z-index 1
```

Comment:
why this z-index? What is the purpose? You should use our css's var to handle zIndex https://github.com/cozy/cozy-ui/blob/67485ca3a6db28dd4ba64d2c91188c21b00034ce/stylus/settings/z-index.styl#L55-L69

Reply:
Will use a variable. The z-index is necessary for the header drop shadow to spread over the actual editor area (otherwise the editor area will erase the shadow).